app/assets/stylesheets/small.scss

```diff
@@ -7,6 +7,7 @@ body.small {
   input[type="submit"],
   input[type="text"] {
     -webkit-appearance: none;
+
```

Comment:
Please don't add random blank lines that aren't related to other changes (this isn't the only one).